docs/mp/health.adoc

```diff
@@ -163,18 +164,18 @@ The example below will change the root path.
 [source,properties]
 .Create a file named `microprofile-config.properties` in the `resources/META-INF` directory with the following contents:
 ----
-health.web-context=myhealth  #<1>
+health.endpoint=/myhealth  #<1>
 ----
-<1> The web-context specifies a new root path for the health endpoint.
+<1> The `endpoint` setting specifies the root path for the health endpoint.
 
 == Examples
 
-Generate Helidon MP Quickstart project following these xref:guides/quickstart.adoc[Instruction]
+Generate Helidon MP Quickstart project following these xref:guides/quickstart.adoc[instructions].
 
 === Using the Built-In Health Checks
 
-Helidon has a set of built-in health checks that are enabled to report various
-health check statuses that are commonly used:
+Helidon has a set of built-in health checks which report various
```

Comment:
Helidon has a set of built-in health checks that can report various

Reply:
Fixed.